config/locales/en.yml

```diff
@@ -88,10 +88,10 @@ en:
             amount:
               blank: Enter income the dependant gets
               blank_when_many: Enter income dependant %{position} gets
-              dependant_monthly_income_too_high: Dependant income must be less than £338.90 per month. If you have counted someone as a dependant and they have income of £338.90 or more per month, you must go back and remove them as a dependant to continue with this check.
-              dependant_monthly_income_too_high_when_many: Dependant %{position} income must be less than £338.90 per month. If you have counted someone as a dependant and they have income of £338.90 or more per month, you must go back and remove them as a dependant to continue with this check.
-              dependant_monthly_equivalent_income_too_high: Dependant income must be less than the equivalent of £338.90 per month. If you have counted someone as a dependant and they have income equivalent to £338.90 or more per month, you must go back and remove them as a dependant to continue with this check.
-              dependant_monthly_equivalent_income_too_high_when_many: Dependant %{position} income must be less than the equivalent of £338.90 per month. If you have counted someone as a dependant and they have income equivalent to £338.90 or more per month, you must go back and remove them as a dependant to continue with this check.
+              dependant_monthly_income_too_high: Dependant income must be less than %{limit} per month. If you have counted someone as a dependant and they have income of %{limit} or more per month, you must go back and remove them as a dependant to continue with this check.
+              dependant_monthly_income_too_high_when_many: Dependant %{position} income must be less than %{limit} per month. If you have counted someone as a dependant and they have income of %{limit} or more per month, you must go back and remove them as a dependant to continue with this check.
+              dependant_monthly_equivalent_income_too_high: Dependant income must be less than the equivalent of %{limit} per month. If you have counted someone as a dependant and they have income equivalent to %{limit} or more per month, you must go back and remove them as a dependant to continue with this check.
+              dependant_monthly_equivalent_income_too_high_when_many: Dependant %{position} income must be less than the equivalent of %{limit} per month. If you have counted someone as a dependant and they have income equivalent to %{limit} or more per month, you must go back and remove them as a dependant to continue with this check.
```

Comment:
Something not quite right here, with error message, as it can't find the `%{limit}` variable.
![Screenshot 2024-03-21 at 12 44 42](https://github.com/ministryofjustice/laa-check-client-qualifies/assets/76905544/70ad942e-8e1a-4234-83fc-b61e63b5b125)